Adafruit_TCS34725.cpp

```diff
@@ -294,6 +294,30 @@ void Adafruit_TCS34725::getRawDataOneShot(uint16_t *r, uint16_t *g, uint16_t *b,
   disable();
 }
 
+/*!
+ *  @brief  Read the RGB color detected by the sensor.
+ *  @param  *r
+ *          Red value
```

Comment:
please add here, "normalized to 0-255" - same for g and b

Reply:
fixed :)